src/components/Card/Card.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Styled from './styled';
```

Comment:
You might want to rename the file to Card.styled.js so when you search for the styles in your text editor don't have to look up a thousand different styled.js files.
Also, instead of importing everything into Styled, import each styled component separately: 

`import { StyledCol, StyledCard } from './Card.styled.';`